ivadomed/main.py

```diff
@@ -180,14 +179,16 @@ def cmd_train(context):
     if context['retrain_model'] is None:
         if HeMIS:
             model = models.HeMISUnet(modalities=context['contrast_train_validation'],
+                                     out_channel=len(context["target_suffix"]) + 1,
```

Comment:
Why +1?

Reply:
It is for the background class since I use softmax I need this extra output I'll leave a comment in the code :)